src/layout/drawer/index.tsx

```diff
@@ -7,9 +7,12 @@ import styled from 'styled-components';
 import LogoImg from 'assets/CTB_Text_black.png';
 import { useAuthSignOutAndRedirect } from 'auth/hooks';
 
+import { useSnackBars } from '../../components/snackbar';
```

Comment:
jak importujemy z /src/components to nie chcemy relatywnej ścieżki, popraw też w innych miejscach

```suggestion
import { useSnackbars } from 'components/snackbar';
```